CSharp/intelligence-LUIS-Translator/Microsoft.Bot.Builder.AI.Translation/RequestBuilder/TranslatorRequestBuilder.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Microsoft.Bot.Builder.AI.Translation.Model;
+using Newtonsoft.Json;
+[assembly: InternalsVisibleTo("Microsoft.Bot.Builder.AI.Translation.Tests")]
+
+namespace Microsoft.Bot.Builder.AI.Translation.RequestBuilder
+{
+    /// <summary>
+    /// Provides http requests needed for translation and language detection.
+    /// </summary>
+    internal class TranslatorRequestBuilder : IRequestBuilder
```

Comment:
I guess it's better to name it just RequestBuilder since it;s already inside this namespace : 
Microsoft.Bot.Builder.AI.Translation.RequestBuilder

Reply:
I initially named it RequestBuilder but I then had to use it like RequestBuilder.RequestBuilder ..
also I've read [that](https://blogs.msdn.microsoft.com/ericlippert/2010/03/09/do-not-name-a-class-the-same-as-its-namespace-part-one/)